test/ut/sdk/test_tuner_command_channel.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import atexit
+from dataclasses import dataclass
+import importlib
+import json
+import os
+from pathlib import Path
+from subprocess import Popen, PIPE
+import sys
+import time
+
+from nni.runtime.tuner_command_channel.websocket import WebSocket
+
+# A helper server that connects its stdio to incoming WebSocket.
+_server = None
+_client = None
+
+_command1 = 'T_hello world'
+_command2 = 'T_你好'
+
+## test cases ##
+
+def test_connect():
+    global _client
```

Comment:
Does this work for pytest?

Reply:
Theoretically yes, but Windows test still have problem...